src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1329,6 +1329,10 @@
     <Field Name="Expanded" Type="bool"/>
     <Field Name="ArgsToParamsOpt" Type="ImmutableArray&lt;int&gt;" Null="allow"/>
     <Field Name="ResultKind" PropertyOverrides="true" Type="LookupResultKind"/>
+    
+    <!-- BinderOpt and UseSetterForDefaultArgumentGeneration are added as a temporary solution for IOperation implementation and should probably be removed in the future -->
```

Comment:
Thanks for the context.
Do we have an issue tracking removing this in the future?

---
In reply to: [126044205](https://github.com/dotnet/roslyn/pull/20689#discussion_r126044205) [](ancestors = 126044205)